Rakefile

```diff
@@ -5,6 +5,13 @@ namespace :db do
     task :create do
       sh "createdb travis_#{env}" rescue nil
       sh "psql -q travis_#{env} < #{Gem.loaded_specs['travis-migrations'].full_gem_path}/db/structure.sql"
+
+      # logs database
+      require 'sequel'
+      Sequel.extension(:migration)
+      db = Sequel.connect(:adapter => 'postgres', :database => "travis_#{env}")
```

Comment:
We will want this to be the logs database setup in our database.yml (which we will want to add) so people can specify the database name they want (i.e.: I run Travis-logs as a separate DB locally like production does)